app/models/user.rb

```diff
@@ -55,4 +55,16 @@ def authenticate(password)
   def locked?
     self.locked_at.present?
   end
+
+  def current_organization
+    self.organizations.find(self.current_organization_id)
+  rescue
```

Comment:
@yunixon try not to use `rescue`. It works slow. In this case it's better to check if current_organization_id is blank and if it is return first organization.